### DISCUSSION TITLE
Group Assignment #2 - Memo 📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#11] [Assignments]
## Due 4/27

Have one person comment on this post to submit the link to your [group assignment #2](https://albertkun.github.io/23S-ASIAAM-191A/assignments/week2/group_assignment/) memo with your **group name** in this thread.

### For example

Awesome Group - [https://docs.google.com/document/d/1OWUUxebPNsqw4K97gznvTkGLEr6QDAbXWqX2BGvernI/copy](https://docs.google.com/document/d/1OWUUxebPNsqw4K97gznvTkGLEr6QDAbXWqX2BGvernI/copy)

**Comment by paige-lee** [comment]
Stop Asian Hate: https://github.com/ariscorpus/stopasianhate/blob/main/AS%20AM%20191A%20-%20Project%20%232.pdf

**Comment by zacharypakin** [comment]
College/Career Prep in High Schools: https://docs.google.com/document/d/1SlxKN--ncXRuaQKxXF002cXKd_axPdxvhcRCj5rBJLs/edit?usp=sharing

**Comment by cassiuscarandang** [comment]
Housing Security 4 All: https://docs.google.com/document/d/1FT8f9JGi9b3lmb5S8jR_yNAL9k9yCb7yiGSRtZOYTqY/edit?usp=sharing

**Comment by catherinehhu** [comment]
Oil Drilling: https://docs.google.com/document/d/1PNL_RTv_8SrENZGaJAVXmrxEg7cr3xFCVnmvfK0y3qk/edit

**Reply by albertkun** [reply]
Just getting around the reviewing the memos and I don't have access to it @catherinehhu 

![image](https://user-images.githubusercontent.com/8574425/235299173-bb3f6e6f-ad37-4c29-a883-51568d0cf425.png)

**Comment by kmtsujimoto** [comment]
Federally Qualified Healthcare Centers: https://docs.google.com/document/d/1ydP-jpP1JHqKkHpxV2v1MpcVio6JViphi8Da7DibRXg/edit?usp=sharing

**Comment by SageLuong** [comment]
Westwood Reimagined: Mapping for Human-centric Infrastructure -https://docs.google.com/document/d/1DESNRm_XL7CiTPXDSH4kTliBQYSsVXHVIHh4usJMH5g/edit?usp=sharing

**Comment by emai14** [comment]
Educational Attainment: https://github.com/rebeccaksuen/Educational-Attainment/blob/master/Memo.pdf

**Comment by danigold1020** [comment]
Ocean Pollution: https://docs.google.com/document/d/1zgN4oBwbOEASws4noyHNEINXmB_a6VSdyEVYBMFaF5U/edit?usp=sharing

**Comment by alanamalika** [comment]
Southeast Asian Retention at UCLA:
[https://docs.google.com/document/d/1l9W414V8UDmaLfZBaI6EVYt5nhL2Q27mAYzRwXqQ_4o/edit](https://docs.google.com/document/d/1l9W414V8UDmaLfZBaI6EVYt5nhL2Q27mAYzRwXqQ_4o/edit)

**Comment by qatle** [comment]
Gender Inequality in Transportation: docs.google.com/document/d/1olcr3Z4hpixXaZKbzb3oDmfF1uPUQZj2akojj_KKlI4/edit?usp=sharing

**Reply by albertkun** [reply]
@trinhqale I do not have access to your doc: 
![image](https://user-images.githubusercontent.com/8574425/235392720-1df6ef21-0ae5-4ea9-9ca7-cdf147f46de9.png)

**Reply by qatle** [reply]
I'm sorry I got the wrong link: https://docs.google.com/document/d/1olcr3Z4hpixXaZKbzb3oDmfF1uPUQZj2akojj_KKlI4/edit?usp=sharing
